src/facades/wordpress.php

```diff
@@ -12,9 +12,21 @@ function whip_wp_check_versions( $requirements ) {
 			return;
 		}
 
+		global $wp_version;
+
+
 		$config  = include dirname( __FILE__ ) . '/../configs/default.php';
 		$checker = new Whip_RequirementsChecker( $config );
 
+		$dismisser = new Whip_MessageDismisser( $wp_version, new Whip_WPDismissOption() );
+
+		$dismissListener = new Whip_WPMessageDismissListener( $dismisser );
```

Comment:
The dismiss listener should only be added further down below if a message is indeed being presented.